tests/common/test_windowedsum.py

```diff
@@ -0,0 +1,59 @@
+#
+# This source file is part of the EdgeDB open source project.
+#
+# Copyright 2011-present MagicStack Inc. and the EdgeDB authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from __future__ import annotations
+from typing import *  # NoQA
+
+import unittest
+import unittest.mock
+
+from edb.common.windowedsum import WindowedSum
+
+
+class FrozenTime:
```

Comment:
nit: it's not frozen, it's "ControlledClock" or something.

Reply:
Renamed as ManualClock